direct/algorithms/mri_algorithms.py

```diff
@@ -0,0 +1,181 @@
+# coding=utf-8
+# Copyright (c) DIRECT Contributors
+
+"""This module contains mathematical optimization techniques specific to MRI."""
+
+from typing import Any, Callable, Dict
+
+import numpy as np
+import torch
+from torch import nn
+
+from direct.algorithms.optimization import MaximumEigenvaluePowerMethod
+from direct.data.transforms import crop_to_acs, view_as_complex, view_as_real
+from direct.types import KspaceKey
+from direct.utils import DirectModule
+
+
+class EspiritCalibration(DirectModule):
+    """Estimates sensitivity maps estimated with the ESPIRIT calibration method as described in [1]_.
+
+    We adapted code for ESPIRIT method adapted from [2]_.
```

Comment:
Why does this not derive from `Algorithm`?

Reply:
This is a transform-style class